cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -173,8 +173,7 @@ module ice_grid
          use_bathymetry, & ! flag for reading in bathymetry_file
          pgl_global_ext    ! flag for init primary grid lengths (global ext.)
 
-      logical (kind=log_kind), &
-         dimension (:,:,:), allocatable, public :: &
+      logical (kind=log_kind), dimension (:,:,:), allocatable, public :: &
          tmask  , & ! land/boundary mask, thickness (T-cell)
```

Comment:
As long as you're changing grid variable names, you might as well make them all consistent.